modules/integration/src/test/scala/scala/cli/integration/SipScalaTests.scala

```diff
@@ -107,6 +107,20 @@ class SipScalaTests extends ScalaCliSuite {
       }
     }
 
+  if (TestUtil.isNativeCli)
+    test(s"usage instruction should point to scala when installing by cs") { // https://github.com/VirtusLab/scala-cli/issues/1662
+      TestInputs.empty.fromRoot {
+        root => // cs install binaries under .app-name.aux and scala-cli should drop .aux from progName
+          val binary       = "scala".prepareBinary(root)
+          val csBinaryName = root / ".scala.aux"
+          os.move(binary, csBinaryName)
+          val output   = os.proc(csBinaryName, "test", "--usage").call(check = false).out.text()
+          val usageMsg = TestUtil.removeAnsiColors(output)
+          expect(usageMsg.contains(" scala "))
+          expect(!usageMsg.contains(".scala.aux"))
```

Comment:
What is the entire message? I think it should be exactly:

> Usage: scala test [options]

Is this possible?

Reply:
If `case-app` doesn't change the message for usage options, it should be exactly `Usage: scala test [options]`.

But for now, it should be exactly `Usage: scala test [options]`, so I modify integration test.